packages/fx-core/resource/package.nls.json

```diff
@@ -802,6 +802,11 @@
   "error.core.appIdNotExist": "Cannot find app id: %s. Either your current M365 account does not have permission, or the app has alredy been deleted.",
   "driver.apiKey.description.create": "Create an API key on Developer Portal for authentication in Open API spec.",
   "driver.aadApp.apiKey.title.create": "Creating API key...",
+  "driver.apiKey.description.update": "Update an API key on Developer Portal for authentication in Open API spec.",
+  "driver.aadApp.apiKey.title.update": "Updating API key...",
```

Comment:
In title, we don't need ellipses (...) as they show continuation.

Reply:
Yes it is the message shown in the progress bar.